deploy/FlapperInit.sol

```diff
@@ -62,67 +63,56 @@ interface FarmLike {
     function setRewardsDuration(uint256) external;
 }
 
-interface FlapperCaller {
-    function file(bytes32, address) external;
-}
-
 struct FlapperUniV2Config {
-    uint256 hop;
     uint256 want;
     address pip;
-    uint256 hump;
-    uint256 bump;
-    address caller;
+    address pair;
+    address splitter;
     bytes32 chainlogKey;
 }
 
 struct SplitterConfig {
+    uint256 hump;
+    uint256 bump;
+    uint256 hop;
     uint256 burn;
     uint256 rewardsDuration;
+    address farm;
```

Comment:
I think it was noted already, but we can probably just use `hop` as rewardsDuration.

Reply:
Done in c5b843